docs/release_notes/0.87.0.md

```diff
@@ -2,5 +2,10 @@
 
 ## Features
 
-- Added validation check for zones before creating cluster (#4837)
+- Add validation check for zones before creating cluster (#4837)
+
+## Bug Fixes
+
+- Fix missing stack name in case only the stack is provided. (#4894)
```

Comment:
@Skarlso I feel like this could probably change :joy: :see_no_evil:

Reply:
Yeah, possible. :D